ci/association.hoc

```diff
@@ -1,9 +1,10 @@
 objref f
 f = new File()
-f.ropen("temp.txt")
+f.ropen("association.hoc.out")
 f.close()
 f.unlink()
```

Comment:
Shouldn't the close and unlink be done only if f.ropen returned non-zero

Reply:
I will adjust. Haven't had any problems with it though (that file doesn't exist when the CI is run)

Reply:
I'm kind of sorry I mentioned it. f.close is safe as it internally checks that FILE* is non-null before calling close.
However, I see that the internal unlink will set ENOENT if the file does not exist.